### PR TITLE
Adding armor stuff to the coc outfit.

### PR DESCRIPTION
**Pull request by Buzzardman2**
## About The Pull Request
Gives paps outfit armor and the ability to wear gun i think.

## Why It's Good For The Game

Pap will beat me with a mop if i don't do this :p

## Changelog
:cl:
add: Changed the armor to be based on armor so pap can actually use it properly.
/:cl:

### CODE REVIEW

File: ModularBungalow/clothing/coat.dm [hunk 1]
```diff
@@ -99,7 +99,7 @@
 	inhand_icon_state = "armor"
 	dog_fashion = null
 
-/obj/item/clothing/suit/CoCoat
+/obj/item/clothing/suit/armor/centcom/CoCoat
```
Comment (EgorDinamit) [inline ModularBungalow/clothing/coat.dm:102]: You do realize that it makes it as strong as death squad armor, if not more?
No. We certainly don't have to give some shitty janitor coat that much armor.

Comment (thevandie) [inline ModularBungalow/clothing/coat.dm:102]: better idea, make it " /obj/item/clothing/suit/armor/captain/CoCoat " so it uses cap armor values. The armor centcom would be the buggy old green cap hardsuit thing

Comment (Kitsunemitsu) [reply ModularBungalow/clothing/coat.dm:102]: Usually these Admin armor sets aren't seen.
Like ever. Even admins dont' use them during normal play

Comment (EgorDinamit) [reply ModularBungalow/clothing/coat.dm:102]: Oh, so that's not available normally? If so - then it's fine.

Comment (Buzzardman2) [reply ModularBungalow/clothing/coat.dm:102]: Okay i can switch it to that no problem.

**Review by EgorDinamit — APPROVED** [review]
Powercreep janitor armor that Pap will wear when nobody's looking.
For all intents and purposes - approved.

**Pull request merged**